src/db/file.rs

```diff
@@ -13,20 +12,16 @@ use std::path::{Path, PathBuf};
 
 /// Store all files in a directory and return [[mimetype, filename]] as Json
 ///
-/// If there is an S3 Client configured, store files into an S3 bucket;
-/// otherwise, stores files into the 'files' table of the local database.
-///
```

Comment:
Why'd this comment disappear?

Reply:
So, that comment is going to be false in a future PR (I want to remove autodetection), dunno why I removed it now.

Reply:
Added the comment back.